src/Mutation/Mutation.php

```diff
@@ -164,6 +166,16 @@ public function getAllTests(): array
         return $this->tests;
     }
 
+    /**
+     * Overall time needed to run known tests for a mutation, excluding dependencies.
+     */
+    public function getTimeToTest(): float
```

Comment:
```suggestion
    public function getTotalTestExecutionTime(): float
```

Reply:
This is a tricky moment. We can't say it is execution time because one may think this is actual test execution time, where it isn't. For example, this timing does not include dependent tests, therefore actual execution time might be larger.

Reply:
Is there a way we could ignore those dependent tests when executing them?

Reply:
[I don't think this is feasible.](https://phpunit.readthedocs.io/en/9.0/writing-tests-for-phpunit.html#writing-tests-for-phpunit-examples-stacktest2-php)

Reply:
`getTestExecutionTime` then? `time to test` sounds a bit weird, and I think keeping a comment to mention it does not count the dependent tests is fine

Reply:
Renamed into `getNominalTestExecutionTime`